shotwell-desktop.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/misc/shotwell.desktop.in b/misc/shotwell.desktop.in
+index aa1ede2..2800ebf 100644
+--- a/misc/shotwell.desktop.in
++++ b/misc/shotwell.desktop.in
+@@ -5,7 +5,7 @@ GenericName=Photo Manager
+ Comment=Organize your photos
+ # Translators: Search terms to find this application. Do NOT translate or localize the semicolons! The list MUST also end with a semicolon!
+ Keywords=album;camera;cameras;crop;edit;enhance;export;gallery;image;images;import;organize;photo;photographs;photos;picture;pictures;photography;print;publish;rotate;share;tags;video;facebook;flickr;picasa;youtube;piwigo;
+-Exec=shotwell %U
++Exec=shotwell-wrapper %U
```

Comment:
You could use something like this https://github.com/flathub/com.github.z.Cumulonimbus/blob/785266a8ec0c2ebb73eea05571518238e193eb0c/com.github.z.Cumulonimbus.json#L32 instead of patching the desktop file. Could be added as `post-install` as you seem to be using it already on the manifest.

Reply:
The wrapper needs to be removed anyway so it doesn't matter.